README.md

```diff
@@ -167,4 +144,31 @@ You can also use the `usePlaidEmitter` hook in react functional components:
   })
 ```
 
+# Version information
+
+## Versions and release candidates
+
+We create release candidates (e.g. 7.0.0-rc1) as beta previews for developers. These are helpful for customers who either are 1. waiting for a specific fix or 2. extremely eager for specific features. They do not hold the same quality guarantee as our official releases, and should NOT be used in production. The official releases come ~2 weeks after the first release candidate (rc1).
+
+The latest stable version is the highest version without the suffix `-rcX`.
+
+## Updating from previous versions.
+
+When upgrading from a previous major version of this library, see the following notes for additional instructions:
+
+- Upgrading [pre 5.x](./upgrade_notes)
+- Upgrading [from 5.x onwards][upgrading]
+
+# Version compatibility
+| React Native SDK | Android SDK | iOS SDK | Status |
+|---|---|---|---|
+| 7.x.x | [3.2.0+]      | >=2.0.7  | Active     |
```

Comment:
Maybe this status should be "In Development" instead of "Active

Reply:
Great point!